.github/workflows/wheels.yml

```diff
@@ -8,29 +8,54 @@ on:
 
 jobs:
 
-  build-wheel:
-    runs-on: ${{ matrix.os }}
+  build-wheel-linux:
+    runs-on: ubuntu-18.04
+    container: pytorch/manylinux-cpu
     strategy:
       matrix:
-        py_ver: [ "3.7", "3.8", "3.9" ]
-        os: ["ubuntu-18.04", "macos-latest"]
+        python_abi: [ "cp37-cp37m", "cp38-cp38", "cp39-cp39" ]
+    steps:
+      - name: Checkout functorch
+        uses: actions/checkout@v2
+      - name: Install PyTorch Nightly
+        run: |
+          export PATH="/opt/python/${{ matrix.python_abi }}/bin:$PATH"
+          python3 -mpip install --pre torch>=1.12.0.dev -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+      - name: Build wheel
+        run: |
+          export PATH="/opt/python/${{ matrix.python_abi }}/bin:$PATH"
+          python3 -mpip install wheel
+          python3 setup.py bdist_wheel
+          # NB: wheels have the linux_x86_64 prefix, need to be manually renamed
+      - name: Upload wheel as GHA artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: functorch-linux.whl
+          path: dist/*.whl
+
+  build-wheel-mac:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python_version: [ "3.7", "3.8", "3.9" ]
     steps:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.py_ver }}
+          python-version: ${{ matrix.python_version }}
```

Comment:
@malfet -- the comment here https://github.com/actions/setup-python/issues/26 is a bit troubling. Ideally our binaries would be compatible with the 10.9 SDK (like the rest of PyTorch), but the thread suggests that github actions sets up Python to be compatible with the 10.14 SDK. Does this mean we have to install python a different way? (e.g. conda like how torchvision and torchaudio are doing it).